libs/ui/src/test_mode.tsx

```diff
@@ -50,3 +52,42 @@ export function TestMode(): JSX.Element {
     </ThemeProvider>
   );
 }
+
+const TestModeCard = styled.div`
+  font-size: ${(p) => p.theme.sizes.fontDefault}px;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+  padding: 0.125em 0.5em;
+  border: ${(p) => p.theme.sizes.bordersRem.thin}em solid
```

Comment:
For my learning - Is `em` the standard approach in `libs`? Maybe to ensure we inherit the parent's root sizes?